playbook.md

```diff
@@ -827,31 +819,92 @@ TODO. This section to cover:
 * The probation period, meeting and expectations
 * ...more things.
 
-### The buddy system
```

Comment:
I agree. There's a card for it in the company retro board, so let's track it there.

https://trello.com/c/byNvuPgD/92-get-feedback-on-buddy-system-from-the-team